main.c

```diff
@@ -423,6 +432,50 @@ void closeAudio()
 }
 #endif
 
+#if __APPLE__ && (TARGET_IPHONE_SIMULATOR || TARGET_OS_IPHONE)
```

Comment:
It seems this doesn't do anything. Are some paths TODOs? If so, comments should be added. Otherwise, I'd remove the cases.

Maybe we want to do something about backgrounding at some point, like saving all state, in case we get killed – or are we guaranteed to get "terminating" in this case? In any case, we don't have full state save/restore yet, so that would be a TODO.

Reply:
:+1: